localization/english/dvg_flavor_russia_l_english.yml

```
@@ -1,14 +1,17 @@
+  je_russian_anarchy:0 "The Endless Troubles"
+  je_russian_anarchy_reason:0 "Russia has tried to unite several times, and all attempts have failed. A continued conspiracy between the king in Warsaw and the king in Copenhagen have kept the Russians weak and in a permanent state of near anarchy. However, as both states are doomed to suffer this century: it might finally be possible to unite the scattered Russian Princes and finally create Russia.\n\n #BOLD Russian Anarchy#! will trigger on one of the following conditions:\n - #BOLD Nationalism#! is discovered and #BOLD Green Prince#! resolved\n - The year is 1850\n - Scandinavia or Poland is conquering Russian lands"
+  has_russian_anarchy: "#BOLD Russian Anarchy#! triggered"
   united_muscovy: "Unite central Russia under one nation."
   je_mus_southern_expansion:0 "Southern Expansion"
   je_mus_southern_expansion_reason:0 "The South was always caught in a chaotic swirl that did not characterize our homeland in central Russia. The South is a diverse and wild land. With our newly found unity, the South appears less threatening each day, and conquering and taming it may bear fruit for our nation. We should expect a struggle, but Russians march towards the Black Sea is inevitable, for the land belongs to holy Russia alone!"
   je_mus_northern_expansion:0 "Northern Expansion"
   je_mus_northern_expansion_reason:0 "Novgorod stood as an abomination in our eyes for a long time, with a government of a mangled republic combined with a corrupt principality. Serving as the lap dogs of Scandinavia has not given Novgorod the prosperity and power to block our rightful path for the restoration and unification of the Russian lands. As our holdings in central Russia crystalize, we can finally march north and knock the arrogant merchants of Novgorod off their throne, and take it for ourselves and for all of Russia."
   je_rus_kazan_expansion:0 "Left bank of the Volga"
   je_rus_kazan_expansion_reason:0 "For centuries the Tatars have ravaged the lands of the holy Rus, robbing, pillaging and enslaving our people. Not anymore, our countrymen say, for now Russia is united and strong. Our borders are well defined on the western and northern frontiers, but the savage nomads of the East threaten the security of our realm, and to guarantee that security we must secure the left bank of the mighty Volga river. From there, the ancient threat of the Tatars will evaporate and Russia will remain ascendant on the eastern frontier."
+  je_smo_reconquest:0 "Retake rightfull lands"
+  je_smo_reconquest_reason:0 "We have to retake back our our rightfull lands!"
```

```suggestion
  je_smo_reconquest_reason:0 "We have to retake back our our rightful lands!"
```

Same with the line above